olive/passes/openvino/conversion.py

```diff
@@ -19,31 +19,47 @@ class OpenVINOConversion(Pass):
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
         return {
             "input": PassConfigParam(
-                type_=List[Tuple],
+                type_=List,
                 required=False,
                 description=(
-                    "Input can be set by passing a list of tuples. "
-                    "Each tuple should contain input name and optionally input type or input shape."
+                    "Set or override shapes for model inputs. "
+                    "It configures dynamic and static dimensions in model inputs "
+                    "depending on your inference requirements."
                 ),
             ),
-            "input_shape": PassConfigParam(
-                type_=List[int],
+            "example_input": PassConfigParam(
```

Comment:
since torch tensor or other inputs cannot be serialized, should this be made into a function or string name paramater instead. like for `dummy_input_func` or `dataloader_func`